pulp_rpm/app/advisory.py

```diff
@@ -256,7 +256,7 @@ def _copy_update_collections_for(advisory, collections):
     new_collections = []
     with transaction.atomic():
         for collection in collections:
-            uc_packages = collection.packages.all()
+            uc_packages = list(collection.packages.all())
```

Comment:
Problem: The queryset on line 264 was evaluating as empty

Our hypothesis is that maybe the queryset previously kept a record of the original PK, and the new behavior is that it uses the current object's PK, which is a completely new object.